cypress/integration/host/host_buttons.ts

```diff
@@ -14,6 +14,14 @@ describe("Host page restart jasper and update host status buttons", () => {
     cy.dataCy("restart-jasper-button").click();
     cy.get(popconfirmYesClassName).click();
     cy.dataCy("toast").should("exist");
+    cy.get(`[aria-label="Close Message"]`).click();
+  });
+
+  it("Should show a toast when host is reprovisioned", () => {
+    cy.dataCy("reprovision-button").click();
+    cy.get(popconfirmYesClassName).last().click();
```

Comment:
I'm not sure why I need this extra `last()` call for the reprovision test but not the restart jasper test. The e2e tests won't pass without it.

I also tried replacing both of these calls with `cy.contains("button", "Yes").click()`, but similarly, this passes for the first call and fails for the second call.

Reply:
Wanna dive down the rabbit hole of components mounting on to the dom when you first open the tooltip but not unmounting? 
The tooltip is mounting on to the dom when you click on the button in the previous test. 
When the tooltip closes it still exists on the dom. Since you are selecting it based off of a classname selector it is finding the first tooltip you clicked. That still exists but just isn't visible.